zhaquirks/philips/hue_light.py

```diff
@@ -169,3 +169,36 @@
     .replaces(PhilipsHueLightCluster, endpoint_id=11)
     .add_to_registry()
 )
+
+(
+    QuirkBuilder()
+    .applies_to(SIGNIFY, "LCD003")
+    .friendly_name(
+        model="Hue Akari Downlight",
+        manufacturer="Philips",
+    )
+    .replaces(PhilipsHueLightCluster, endpoint_id=11)
+    .add_to_registry()
+)
+
+(
+    QuirkBuilder()
+    .applies_to(PHILIPS, "1743530P7")
+    .friendly_name(
+        model="Hue Discover Outdoor Floodlight RGBCCT",
```

Comment:
And here, I'd go for this:
```suggestion
        model="Hue Discover outdoor floodlight RGBCCT",
```
or
```suggestion
        model="Hue Discover outdoor floodlight",
```

The RGBCCT is a bit weird, since we don't include that in other names. Maybe we could only include "white" in-place of RGBCCT for the white-only models? Or "color" for this?